src/pages/settings/company/components/Defaults.tsx

```diff
@@ -83,6 +83,23 @@ export function Defaults() {
             </Element>
           )}
 
+          <Element leftSide={t('expense_payment_type')}>
+            <SelectField
+              value={companyChanges?.settings?.default_expense_payment_type_id}
+              onChange={handleChange}
+              id="settings.default_expense_payment_type_id"
+            >
+              <option value="0"></option>
```

Comment:
SelectField comes with a `withBlank` prop that injects the empty field. Does that not fit this case?

Reply:
We couldn't use it because we needed a blank option value of `'0'`, but the blank option only had an empty string value. So, I made improvements to the `SelectField` component by adding a `blankOptionValue` and improved the implementation of SelectFields on the `Defaults` component. I hope you find my solution agreeable.